README.md

```diff
@@ -4,7 +4,9 @@
 
 All clients are listed under language specific subfolders of [clients](./clients)
 
-The path follows the pattern: ``clients/{language}/github.com/{owner}/{repository}.json``
+The path follows the pattern: ``clients/{language}/github.com/{owner}/{repository}.json``.
+Note that the ``{language}`` component of the above path is the path-safe representation
```

Comment:
I don't think this needs "Note that". It's simply a continuation of the description of the path structure.

```suggestion
The ``{language}`` component of the path is the path-safe representation
```